src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -12,6 +12,7 @@ Imports Microsoft.CodeAnalysis.Completion.Providers
 Imports Microsoft.CodeAnalysis.CSharp
 Imports Microsoft.CodeAnalysis.CSharp.ExternalAccess.Pythia.Api
 Imports Microsoft.CodeAnalysis.CSharp.Formatting
+Imports Microsoft.CodeAnalysis.CSharp.[Shared].Extensions
```

Comment:
>[Shared] [](http://example.com/codeflow?start=38&length=8)

what is this crazy syntax?

Reply:
it's the VB equivalent of `@class`.  A way to escape a keyword so you can use it as an identifier.  :)